tools/agat/agat.xml

```diff
@@ -302,6 +356,30 @@ check_identical_isoforms: true
             <discover_datasets pattern="__designation_and_ext__" directory="temp_output_distribution_plots/without_isoforms" format="pdf"/>
             <filter>tool['selector'] == 'annotation_statistics'</filter>
         </collection>
+        <data name="sequence_annot_functionnal_report" format="txt" from_work_dir="galaxy_functional_annot/report.txt" label="${tool.name} on ${on_string}: Report">
+            <filter>tool['selector'] =='functional_annotation'</filter>
+        </data>
+        <data name="sequence_annot_functionnal_error" format="txt" from_work_dir="galaxy_functional_annot/error.txt" label="${tool.name} on ${on_string}: Error">
+            <filter>tool['selector'] =='functional_annotation'</filter>
+        </data>
+        <data name="sequence_annot_functionnal_duplicate_blast" format="txt" from_work_dir="galaxy_functional_annot/duplicatedNameFromBlast.txt" label="${tool.name} on ${on_string}: Duplicate name from Blast">
+            <filter>tool['selector'] =='functional_annotation' and blast_adv == 'yes'</filter>
```

Comment:
```suggestion
            <filter>tool['selector'] =='functional_annotation' and blast_adv['options'] == 'yes'</filter>
```

Reply:
have you tried this?

Reply:
Hello, thank you for your feedback.
I tried that, but it doesn't work.

Reply:
I guess it should be `tool['blast_adv']['options']`